conversion/nifti_write.py

```diff
@@ -138,6 +138,7 @@ def nifti_write(inImg, prefix= None):
     hdr_nifti['sform_code']= 2
 
     hdr_nifti['descrip']= 'NRRD-->NIFTI transform by Tashrif Billah'
+    img_nifti = nib.nifti1.Nifti1Image(data, affine=xfrm_nifti, header=hdr_nifti)
```

Comment:
Hi @jhlegarreta , this line is not necessary. Please notice the equality in line 131. That sets the reference to `img_nifti.header`. So any update that is made to `hdr_nifti` after, already updates them in `img_nifti`. You can also test it.

Reply:
If you try to see the header with and without this line, you will see that without this line the header properties that are being set between lines 137-140 are not being written.

Reply:
Run again the script, and double checked. You are right; I need to investigate on my end, as I did not directly use the script, but borrowed the relevant functions.

Reply:
Double checked. Works as expected without the patch set on my end as well. My bad. Sorry for the unnecessary noise @tashrifbillah. Thanks.